apps/src/templates/CountryAutocompleteDropdown.jsx

```diff
@@ -75,6 +75,7 @@ export default class CountryAutocompleteDropdown extends Component {
             placeholder={i18n.searchForCountry()}
             labelKey="value"
             matchPos="start"
+            maxHeight={160}
```

Comment:
Copying over my comment from the [previous PR](https://github.com/code-dot-org/code-dot-org/pull/30060):
> Ideally we'd let the dropdown expand to fill the screen. Can we clear `overflow: auto;` on the modal div, or does that cause other problems?

Reply:
I set the over-flow to "initial" but it didn't fix the country dropdown bug.  This might be due to the overflow property not being a react virtualized select prop (there seems to be a custom option).  See  [react virtualized select](https://github.com/bvaughn/react-virtualized-select)

Since maxHeight is react virtualized select prop, setting the dropdown height to be 80% of the total height of the modal is a solution to this bug.  Thoughts.